src/lib/decorators.ts

```diff
@@ -465,9 +465,10 @@ export function queryAssignedNodes(
         if (nodes && selector) {
           nodes = nodes.filter(
               (node) => node.nodeType === Node.ELEMENT_NODE &&
-                      (node as Element).matches ?
+                      // tslint:disable-next-line:no-any testing existence on older browsers
+                      ((node as any).matches ?
```

Comment:
Since the DOM types include `.matches` you shouldn't need the `as any` cast. Did the old code not type-check?